Demo/src/app/app.component.ts

```diff
@@ -7,4 +7,7 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'Demo';
+  clickMe(){
+    alert((document.getElementById("t1")as HTMLInputElement).value);
```

Comment:
try to avoid javascript code and use angular code

Reply:
ok i will do.